keyboards/latinpad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,86 @@
+
+  /* Keymap _1: (Base Layer) Default Layer
+  
+   * .-----------.
+   * |PGUP | PGDN| 
+   * |-----|-----|-----|-----|
+   * |  7  |  8  |  9  | MO1 |
+   * |-----|-----|-----|-----|
+   * |  4  |  5  |  6  |  +  |
+   * |-----|-----|-----|-----|
+   * |  1  |  2  |  3  |  —  |
+   * |-----|-----|-----|-----|
+   * |  0  |  .  | DEL |Enter|
+   * .-----------------------.
+   */
+     /* Keymap _1: (Base Layer) Default Layer
+  
+   * .------------------.
+   * |NUMLOCK|Caculator | 
+   * |-------|-------|-----|-----|
+   * |RGB_TOG|RGB_MOD|  9  | MO1 |
+   * |-------|-------|-----|-----|
+   * |RGB_M_K|RGB_SAI|  6  |  +  |
+   * |-------|-------|-----|-----|
+   * |RGB_HUI|RGB_VAI|  3  |  —  |
+   * |-------|-------|-----|-----|
+   * |   0   |   .   | DEL |Enter|
+   * .-----------------------.
+   
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT(RGB_MOD, RESET)
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_hasu_5x4(
+	KC_KP_7, KC_KP_8, KC_KP_9, MO(1),
+	KC_KP_PLUS,KC_P6,KC_P5,KC_P4, 
+	KC_P1, KC_P2, KC_P3, KC_KP_MINUS, KC_KP_ENTER,KC_DELETE, KC_PDOT,KC_P0, KC_PGUP, KC_PGDOWN, _______, _______),
+    [1] = LAYOUT_hasu_5x4(RGB_TOG, RGB_MOD, RGB_M_K, RGB_M_X, RGB_SAI, RGB_SAD, RGB_HUI, RGB_HUD, RGB_VAI, RGB_VAD, RGB_SPI, RGB_SPD, RGB_SPI, RGB_SPD, RGB_TOG, KC_TRNS, KC_NUMLOCK, KC_CALCULATOR, _______, _______),
+};
+*/
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_hasu_5x4(
+	KC_KP_7, KC_KP_8, KC_KP_9, MO(1),\
+	KC_KP_PLUS,KC_P6,KC_P5,KC_P4,\
+	KC_P1, KC_P2, KC_P3, KC_KP_MINUS,\
+	KC_KP_ENTER,KC_DELETE, KC_PDOT,KC_P0,\
+	KC_PGUP, KC_PGDOWN,_______, _______),
+    [1] = LAYOUT_hasu_5x4(
+	RGB_TOG, RGB_MOD, RGB_M_K, RGB_M_X,\
+	RGB_SAI, RGB_SAD, RGB_HUI, RGB_HUD,\
+	RGB_VAI, RGB_VAD, RGB_SPI, RGB_SPD,\
+	RGB_SPI, RGB_SPD, RGB_TOG, KC_TRNS,\
+	KC_NUMLOCK, KC_CALCULATOR,_______, _______),
```

Comment:
```suggestion
    [0] = LAYOUT_hasu_5x4(
        KC_PGUP, KC_PGDOWN, _______, _______,
        KC_KP_7, KC_KP_8, KC_KP_9, MO(1),
        KC_KP_PLUS, KC_P6, KC_P5, KC_P4,
        KC_P1, KC_P2, KC_P3, KC_KP_MINUS,
        KC_KP_ENTER, KC_DELETE, KC_PDOT, KC_P0),
    [1] = LAYOUT_hasu_5x4(
        KC_NUMLOCK, KC_CALCULATOR, _______, _______,
        RGB_TOG, RGB_MOD, RGB_M_K, RGB_M_X,
        RGB_SAI, RGB_SAD, RGB_HUI, RGB_HUD,
        RGB_VAI, RGB_VAD, RGB_SPI, RGB_SPD,
        RGB_SPI, RGB_SPD, RGB_TOG, KC_TRNS),
```

Reply:
This reorders your layout to match the physical layout.

Reply:
sir,my layout final row is NUMLOCK CACULATOR